common/source/docs/common-rc-systems.rst

```diff
@@ -64,6 +64,12 @@ FPort/FPort2
 FPort is a bi-directional protocol, using SBus RC in one direction, and serial telemetry in the other. The RC portion can be decoded when attached to an autopilot as if it were SBus, but the embedded telemetry would be lost. See the :ref:`FPort setup documentation<common-FPort-receivers>` for details on connection to one of the autopilots Serial Ports.
 
 
+:ref:`mLRS <common-mlrs-rc>`
+--------------------------
+
+You can connect the RC output pin of your mLRS receiver to the RX pin of a serial port and configure it for CRSF or SBus (The Tx pin is not used).  Or, you can omit the RC connection and configure the mLRS receiver to output RC channels over the telemetry serial connection by injecting RC_CHANNELS_OVERRIDE messages.
```

Comment:
question:how do you get telemetry if only the rx pin is connected??

Reply:
mLRS Rx doesn't handle CRSF telemetry so there is no point in connecting the Tx pin on a port configured for CRSF.  Telemetry on the Rx side of the radio link is always via MAVLink.  mLRS is not intended for use with FC firmware like Betaflight which can't do MAVLink so there is no point in bothering to with CRSF telemetry on the Rx side.  It is expected that a full MAVLink connection is always connected between the Rx and FC.

As stated here, even the CRSF or SBus Rx pin is not really required since mLRS can be configured to deliver the Rc channels to the FC via MAVLink RC_CHANNELS_OVERRIDE.  This is how I always configure mLRS so I can avoid one extra wire and serial port on the FC.  The only reason you really need to connect CRSF or SBus pin is if your use case might require the use of RC_CHANNELS_OVERRIDE message for it's normal purpose where it originates from the GCS.

On the Tx module however, mLRS does use a full CRSF connection.  It translates MAVLink received from the FC via the receiver into CRSF telemetry (sensors) for  use by the radio handset to support things like Yaapu Telemetry.

I didn't discus any of these telemetry details at length on this page because this page is about RC.  Do you have a suggestion for what we could change to prevent readers from wondering about this as you did?  Would it help to explicitly state here that mLRS does not support CRSF telemetry on the Rx end of the radio link?  Or, should I expand on this in common-mlrs-rc?

Reply:
okay without fully studying mlrs it was not apparent that there are two rx ports....the telem port and an sbus/crsf output....why even have a crsf rc only protocol option as rc out?? and cant you mix tx generated overrides with gcs overrides? this really needs a typical wiring diagram fir rx<->fc and tx module to wireless gcs link....I can work on the verbiage on these pages to make things clearer (and fix a few formatting errors)....will take some time for me to generate graphics since one hand is in cast

Reply:
I was asking for suggestions, not for you to solve the problem.  If you think a wiring diagram would help, I'll include one in common-mlrs-rc.

As far as why include the option of CRSF and not just SBUS for RC, I'm not sure what the original motivation was.  But it strikes me that having a choice could be handy since SBUS might require an external inverter depending on the specifics of the FC and Rx.

If the FC receives overrides from both the Rx injected messages based on the RC channels and from the GCS, how would it know which ones to use?  Generating and injecting override messages in the Rx and sending them to the FC is a very handy hack which allows eliminating a wire, but it's a hack which assumes no use of override at the GCS end.  Adding other hacks to squash the RC somewhere in the radio link when the GCS sends overrides is tempting, but then you'd open other policy questions that the radio is not prepared to deal with.  The MAVLink protocol really should define a separate RC message for this use case.

Reply:
OK.  I pushed an update with a small clarification and added a connection diagram for CRSF.  Sorry I missed the formatting error earlier.  Did I miss any others?

Reply:
Also, I checked with Olli and he has given his permission to use this diagram under the CC license which the Wiki uses.